packages/flutter/lib/src/material/banner.dart

```diff
@@ -394,3 +430,51 @@ class _MaterialBannerState extends State<MaterialBanner> {
     );
   }
 }
+
+class _BannerDefaultsM2 extends MaterialBannerThemeData {
+  _BannerDefaultsM2(this.context)
+    : _theme = Theme.of(context),
+      super(elevation: 0.0);
+
+  final BuildContext context;
+  final ThemeData _theme;
+
+  @override
+  Color? get backgroundColor => _theme.colorScheme.surface;
+
+  @override
+  Color? get dividerColor => Theme.of(context).colorScheme.surfaceVariant;
+
+  @override
+  TextStyle? get contentTextStyle => _theme.textTheme.bodyText2;
```

Comment:
I think it should also be  "theme.textTheme.bodyMedium" in M2

Reply:
Yep, I was gonna make this since it just points to  `bodyMedium` anyways.